_targets/build.project.armv7a7-imx6ull

```diff
@@ -115,15 +117,27 @@ b_image_target() {
 	b_mkscript_user "${UPLOAD_SCRIPT[@]}"
 	b_mkscript_preinit
 
-	# Build production image
-	b_log "RAM Image"
-	b_kernel_image "$PREFIX_BOOT/phoenix.img"
-	b_syspage_gen "$PREFIX_BOOT/phoenix.img" "$PLO_SCRIPT_DIR/$NAME_USER_SCRIPT"
-
-	b_log "FLASH Image"
-	b_mkscript_user "${PRIMARY_SCRIPT[@]}"
-	b_kernel_image "$PREFIX_BOOT/phoenix-kernel.img"
-	b_syspage_gen "$PREFIX_BOOT/phoenix-kernel.img" "$PLO_SCRIPT_DIR/$NAME_USER_SCRIPT"
+	if [ "$KERNEL_PLO_BOOT" = "y" ]; then
+		# Build plo
+		make -C plo all
+
+		cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" "$PREFIX_BOOT/plo-ram.img"
```

Comment:
it's a good practice to always use `cp -a` to preserve timestamps (not needed by our build system for these files but external tools might depend on it)